components/responsive-image/image.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>image responsive</title>
+    <link rel="stylesheet" href="image.css">
+    <link rel="stylesheet" href="../common.css">
+</head>
+<body>
+    <h1>Responsive Image</h1>
+    <p>By using below mentioned css classes, you can mirror the below components. </p>
+    
+    <!-- Responsive Image -->
+    <div class="container">
+        <!-- Image -->
+        <div class="image-container">
+            <img class="image-resp" src="../../images/face.jpg" alt="responsive image">
+        </div>
```

Comment:
The image is not responsive, it does not fits the container when the size changes.
Example: I randomly updated the size of container to 500px and the image size did not change accordingly.

<img width="1105" alt="image" src="https://user-images.githubusercontent.com/39650528/153411726-ce051430-b6d6-4394-87d6-3f67a0c9fc10.png">

Reply:
done.